### PR TITLE
Call cover callbacks periodically when moving

### LINKED ISSUE #483: Cover: Auto-stop Task
For setting a specific position for a cover that has no position address 
https://github.com/XKNX/xknx/blob/785d73b850a97a688a3b1a5e8baf34e82685d1b6/xknx/devices/cover.py#L202
we currently use HA to create an updater hook to call `Cover.auto_stop_if_necessary()` periodically to stop the cover again.
https://github.com/XKNX/xknx/blob/785d73b850a97a688a3b1a5e8baf34e82685d1b6/home-assistant-plugin/custom_components/xknx/cover.py#L144

We should implement the auto-stop functionality as a Task in xknx instead of relying on HA to check for updates and call stop then. We get no benefit from doing this over HA now and it would be easier to test.

### PR DESCRIPTION
**Pull request by farmio**
## Description

Cover: call `device_updated_cb` periodically when cover is moving

Closes #483

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] The documentation has been adjusted accordingly
- [ ] The changes generate no new warnings
- [ ] Tests have been added that prove the fix is effective or that the feature works
- [ ] The changes are documented in the changelog
- [ ] The Homeassistant plugin has been adjusted in case of new config options

### CODE REVIEW

**Review by marvin-w — COMMENTED** [review]
Looks good!

**Review by marvin-w — APPROVED** [review]

File: xknx/devices/cover.py [hunk 1]
```diff
@@ -33,12 +33,14 @@
 logger = logging.getLogger("xknx.log")
 
 
+DEFAULT_TRAVEL_TIME_DOWN: Final = 25
+DEFAULT_TRAVEL_TIME_UP: Final = 25
```
Comment (marvin-w) [inline xknx/devices/cover.py:37]: where is this used?

Comment (marvin-w) [reply xknx/devices/cover.py:37]: We should move this to the integration if it's not used in the lib

Comment (farmio) [reply xknx/devices/cover.py:37]: Only in init. (I guess for a lot of tests).

Comment (marvin-w) [reply xknx/devices/cover.py:37]: Ah! Yeah thats fine then :)

**Pull request merged**